src/options-schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "object",
+  "properties": {
+    "test": {},
+    "warningsFilter": {},
+    "extractComments": {},
+    "sourceMap": {
+      "type": "boolean"
+    },
+    "uglifyOptions": {
+      "additionalProperties": true,
+      "type": "object",
+      "properties": {
+        "ecma": {
+          "type": "number",
+          "enum": [5, 6, 7, 8]
```

Comment:
Could make this throw and see if the `ValidationError` is still displayed correctly with this additional info please?